hadoop-hdds/docs/content/security/SecuringS3.zh.md

```diff
@@ -3,7 +3,7 @@ title: "安全化 S3"
 date: "2019-April-03"
 summary: Ozone 支持 S3 协议，并使用 AWS Signature Version 4 protocol which allows a seamless S3
  experience.
-weight: 4
+weight: 5
```

Comment:
Why do we need to change weights?

Reply:
> Why do we need to change weights?

`*.zh.md` and `*.md` are not in the same order on page, just  consistent with `*.md`